### PR TITLE
New resource: azurerm_vpn_server_configuration_policy_group

### PR DESCRIPTION
**Pull request by neil-yechenwei**
This PR is to support new resource `azurerm_vpn_server_configuration_policy_group`.

--- PASS: TestAccVPNServerConfigurationPolicyGroup_basic (271.90s)
--- PASS: TestAccVPNServerConfigurationPolicyGroup_complete (287.59s)
--- PASS: TestAccVPNServerConfigurationPolicyGroup_update (427.01s)
--- PASS: TestAccVPNServerConfigurationPolicyGroup_requiresImport (315.29s)

### CODE REVIEW

File: website/docs/r/vpn_server_configuration_policy_group.html.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+* `policy_member` - (Required) One or more `policy_member` blocks as documented below.
+
+* `is_default` - (Optional) Is this a default VPN Server Configuration Policy Group? Defaults to `false`. Changing this forces a new resource to be created.
+
+* `priority` - (Optional) The priority of this VPN Server Configuration Policy Group. Defaults to `0`.
+
+---
+
+A `policy_member` block supports the following:
```
Comment (katbyte) [inline website/docs/r/vpn_server_configuration_policy_group.html.markdown:64]: member? what does member mean here and would this be better
```suggestion
A `policy` block supports the following:
```

File: website/docs/r/vpn_server_configuration_policy_group.html.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+* `name` - (Required) The name of the VPN Server Configuration Policy member.
+
+* `attribute_type` - (Required) The attribute type of the VPN Server Configuration Policy member. Possible values are `AADGroupId`, `CertificateGroupId` and `RadiusAzureGroupId`.
+
+* `attribute_value` - (Required) The value of the attribute that is used for the VPN Server Configuration Policy member.
```
Comment (katbyte) [inline website/docs/r/vpn_server_configuration_policy_group.html.markdown:70]: is attribute importnat?
```suggestion
* `type` - (Required) The attribute type of the VPN Server Configuration Policy member. Possible values are `AADGroupId`, `CertificateGroupId` and `RadiusAzureGroupId`.

* `value` - (Required) The value of the attribute that is used for the VPN Server Configuration Policy member.
```

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/vpn_server_configuration_policy_group.html.markdown (hunk 2 above)
Comment (neil-yechenwei) [reply website/docs/r/vpn_server_configuration_policy_group.html.markdown:70]: Updated

File: website/docs/r/vpn_server_configuration_policy_group.html.markdown (hunk 1 above)
Comment (neil-yechenwei) [reply website/docs/r/vpn_server_configuration_policy_group.html.markdown]: Updated

**Comment by neil-yechenwei** [comment]
@katbyte , thanks for your comments. I've updated code. Please take another look. Thanks.

**Review by katbyte — APPROVED** [review]
Thanks @neil-yechenwei - LGTM 🦋

**Pull request merged**